samples/bot-conversation/csharp/TeamsApp/env/.env.local

```diff
@@ -16,6 +16,10 @@ AAD_APP_OAUTH_AUTHORITY_HOST=
 TEAMS_APP_TENANT_ID=
 MICROSOFT_APP_TYPE=
 MICROSOFT_APP_TENANT_ID=
+AZURE_SUBSCRIPTION_ID=
+AZURE_RESOURCE_GROUP_NAME=
```

Comment:
It seems there is no `- uses: arm/deploy` in `teamsapp.local.yml.tpl`, so `AZURE_SUBSCRIPTION_ID` and `AZURE_RESOURCE_GROUP_NAME` are unnecessary.